update-timescaledb/update-tsdb-2.md

```diff
@@ -0,0 +1,187 @@
+# Updating TimescaleDB to 2.0 [](update)
+
+Use these instructions to update TimescaleDB 1.x to TimescaleDB 2.0
+
+
+>:WARNING:These instructions are only for upgrading TimescaleDB 1.x to TimescaleDB 2.0
+> If you need to upgrade your existing TimescaleDB 1.x to a newer version in the 1.x
+> release line (eg. 1.7.2 to 1.7.4), please see Update [TimescaleDB 1.x][update-tsdb-1]
```

Comment:
```suggestion
> release line (eg. 1.7.2 to 1.7.4), please see Update [TimescaleDB 1.x][update-tsdb-1].
```

Reply:
Oops! I meant to go back through and look at periods in all docs, especially bulleted lists.  Thanks!